package.json

```diff
@@ -43,20 +43,20 @@
     "webpack-config": "^7.5.0"
   },
   "peerDependencies": {
-    "webpack-dev-server": "^2.11.1",
-    "webpack": "^3.10.0"
+    "webpack": "^3.10.0",
+    "webpack-dev-server": "^2.11.1"
   },
   "devDependencies": {
-    "babel-cli": "^6.26.0",
-    "babel-eslint": "8.2.1",
+    "babel-cli": "^6.24.1",
+    "babel-eslint": "^7.2.3",
     "babel-preset-ca": "^1.1.0",
     "codecov": "3.0.0",
     "commitizen": "^2.9.6",
     "cz-conventional-changelog": "2.1.0",
     "decamelize": "2.0.0",
-    "eslint": "4.16.0",
-    "eslint-config-ca": "3.0.0",
-    "flow-bin": "0.64.0",
+    "eslint": "^3.18.0",
```

Comment:
@alepoletto Do we want to downgrade eslint? Is there some compatibility issue?

Reply:
@samprity I didn't actually downgrade it i've returned it to the original values because my upgrade in the libs caused a compatibility issue for eslint